qubes/vm/adminvm.py

```diff
@@ -52,6 +52,9 @@ class AdminVM(qubes.vm.BaseVM):
     def __str__(self):
         return self.name
 
+    def __lt__(self, other):
+        return self.name < other.name
+
```

Comment:
Dom0 should be sorted before anything else, that is, it should unconditionally `return True`.